### PR TITLE
Add gas price oracle strategies

### LINKED ISSUE #852: Implement gas price oracle
Documentation for https://github.com/api3dao/airnode/issues/1210

### PR DESCRIPTION
**Pull request by vponline**
This adds docs for the three gas price oracle strategies.

### CODE REVIEW

File: docs/airnode/v0.8/grp-providers/guides/build-an-airnode/configuring-airnode.md [hunk 1]
```diff
@@ -127,6 +147,15 @@ chain-specific configuration considerations.
 
 :::
 
+### Considerations: Gas Price Oracle
```
Comment (wkande) [inline docs/airnode/v0.8/grp-providers/guides/build-an-airnode/configuring-airnode.md]: I am going to change this to TOC heading 4.

**Review by wkande — APPROVED** [review]
I made a tiny change for the TOC. Otherwise this looks good and I will merge it now.

File: docs/airnode/v0.8/grp-providers/guides/build-an-airnode/configuring-airnode.md (hunk 1 above)
Comment (vponline) [reply docs/airnode/v0.8/grp-providers/guides/build-an-airnode/configuring-airnode.md]: Sounds good, thanks for checking!

**Comment by martinkolenic** [comment]
This looks fine to me, @wkande. Short and to the point :)

**Pull request merged**